src/ucp/api/ucp.h

```diff
@@ -2051,6 +2100,49 @@ ucs_status_t ucp_rkey_ptr(ucp_rkey_h rkey, uint64_t raddr, void **addr_p);
 void ucp_rkey_destroy(ucp_rkey_h rkey);
 
 
+/**
+ * @ingroup UCP_COMM
+ * @brief Send Active Message
+ *
+ * This routine sends an Active Message to an ep.
+ *
+ * @param [in]  ep          UCP endpoint where the active message will be run
+ * @param [in]  id          Active Message id. Specifies which registered 
+ *                          callback to run.
+ * @param [in]  payload     Pointer to the data to be sent to the target node 
+ *                          for the AM
+ * @param [in]  count       Number of elements to send.
+ * @param [in]  datatype    Datatype descriptor for the elements in the buffer. 
+ * @param [in]  cb          Callback that is invoked upon completion of the data
+ *                          transfer if it is not completed immediately
+ * @param [in]  flags       For Future use
```

Comment:
let add flags if a reply is needed in that case the sender should pack the dest ep ptr